gst/tensor_repopush/tensor_repopush.c

```diff
@@ -0,0 +1,443 @@
+/**
+ * GStreamer
+ * Copyright (C) 2005 Thomas Vander Stichele <thomas@apestaart.org>
+ * Copyright (C) 2005 Ronald S. Bultje <rbultje@ronald.bitfreak.net>
+ * Copyright (C) 2018 Samsung Electronics Co., Ltd.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ */
+
+/**
+ * SECTION: element-tensor_repopush
+ *
+ * Push elemnt to handle tensor repo
+ *
+ * @file	tensor_repopush.c
+ * @date	19 Nov 2018
+ * @brief	GStreamer plugin to handle tensor repository
+ * @see		https://github.com/nnsuite/nnstreamer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "tensor_repopush.h"
+
+/**
+ * @brief Macro for debug mode.
+ */
+#ifndef DBG
+#define DBG (!self->silent)
+#endif
+
+/**
+ * @brief tensor repository
+ */
+extern GstTensorRepo _repo;
+
+/**
+ * @brief Macro for debug message.
+ */
+#define silent_debug(...) \
+    debug_print (DBG, __VA_ARGS__)
```

Comment:
PR #875
Macros in tensot_common.h, debug_print and err_print will be removed.
Please change this.

Reply:
This will be changed in PR #883

Reply:
Now debug_print() is removed. To prevent build error, we have to merge this PR with PR #883.

Reply:
I delete them.